be/src/olap/schema_change.cpp

```diff
@@ -1810,10 +1902,26 @@ OLAPStatus SchemaChangeHandler::_parse_request(TabletSharedPtr base_tablet,
             continue;
         }
 
+        if (materialized_function_map.find(column_name) != materialized_function_map.end()) {
```

Comment:
Maybe the column mapping should be init by the columnPB which comes from the thrift of defineExpr in column.
The AlterReplicaTask does not need the TAlterMaterializedViewParam

Reply:
I think the materialized view is a calculation of an expression. Logic of expr should not be written to the storage layer. Different tablets are irrelevant at the storage level.
So we can refactor this part of the code until the uniform expression evaluates the logic.